src/govuk/components/error-summary/error-summary.yaml

```diff
@@ -120,7 +120,7 @@ examples:
   hidden: true
   data:
     titleText: There is a problem
-    descriptionText: See errors below (▼)
+    descriptionText: See errors below (>)
```

Comment:
I've changed this character `▼` → **`>`** as it wasn't actually something that Nunjucks would escape. We'll now see `>` →  **`&gt;`** which asserts that HTML is escaped